### PR TITLE
Clarifying language from #270

### PR DESCRIPTION
**Pull request by TallTed**
This may need further massaging. I was a bit too slow in reviewing #270.

***
<a href="https://pr-preview.s3.amazonaws.com/TallTed/vc-data-integrity/pull/278.html" title="Last updated on Jul 1, 2024, 9:59 PM UTC (863de8a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/278/f013b59...TallTed:863de8a.html" title="Last updated on Jul 1, 2024, 9:59 PM UTC (863de8a)">Diff</a>

### CODE REVIEW

**Review by Wind4Greg — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1070,32 +1070,32 @@ <h3>Proof Graphs</h3>
...
-similar properties but do not contain an `id` property, or other type of global
-identifier such as a URL, as it is not possible to tell if two unidentified
-objects are expressing information about the same object or not.
+Using these graphs has a concrete effect when performing JSON-LD processing, as
+this properly separates statements expressed in one graph from those in another
+graph. Implementers that limit their processing to other media types, such as
+JSON, YAML, or CBOR, will need to keep this in mind if they merge data from one
+document with data from another, such as when an `id` value string is the same
+in both documents. It is important to not merge objects that seem to have similar
+properties, but which properties do not contain an `id` property or use a global
```
Comment (dlongley) [inline index.html]: Had a hard time parsing this. Maybe this?

```suggestion
properties, but do not contain an `id` property or use a global
```

**Review by dlongley — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html]: Yeah... I think more needs to be tweaked than those words. See below.

File: index.html [hunk 2]
```diff
@@ -1070,32 +1070,32 @@ <h3>Proof Graphs</h3>
...
+identifier type such as a URL, as it is not possible to tell whether two such
+objects are expressing information about the same property of the same object.
```
Comment (TallTed) [inline index.html]: @dlongley @msporny -- What do you think?
```suggestion
in both documents. It is important to not merge objects that seem to have similar
properties, when those objects do not have an `id` property and/or use a global
identifier type such as a URL, as without these, is not possible to tell whether
two such objects are expressing information about the same entity.
```

Comment (msporny) [reply index.html]: LGTM.

Comment (dlongley) [reply index.html]: Agreed.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Purely editorial, merging.

**Pull request merged**